crates/bevy_winit/src/lib.rs

```diff
@@ -49,6 +49,8 @@ use winit::{
     event_loop::{ControlFlow, EventLoop, EventLoopBuilder, EventLoopWindowTarget},
 };
 
+pub use winit::event_loop::EventLoopProxy;
```

Comment:
Nit: it'd be nice to have some documentation pointing users to this pattern somewhere. Maybe on `WinitSettings::desktop_app`? Or maybe there's a better place for it?

Reply:
hi, i added some documentation. I also added it to the bullet points of `Reactive` and `ReactiveLowPower`